interactions/ext/prefixed_commands/context.py

```diff
@@ -80,7 +80,7 @@ async def reply(
         content: Optional[str] = None,
         embeds: Optional[Union[Iterable[Union[Embed, dict]], Union[Embed, dict]]] = None,
         embed: Optional[Union[Embed, dict]] = None,
-        **kwargs,
+        **kwargs: dict,
```

Comment:
I don't think this is correct.
`**` typehints are implicitly `dict[str, T]` where T is your provided hint.  You're saying "Any additional args must be dicts", where I'd normally expect **kwargs to be hinted `Any`

Reply:
In the libs code, I found there are probably 10 different ways kwargs are typehinted, the simplest one was just a basic dict so I went with that to please MkDocs

Reply:
You're misunderstanding [PEP 484](https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values) here.  
This is what you've annotated the parameter as: 
![image](https://github.com/interactions-py/interactions.py/assets/194254/d2466d54-142a-43a1-9e9d-c47915f6294d)

Yes, there are some other places in the library that also do it incorrectly.  You can fix them, or you can not.  But I do not want to merge a PR that makes it worse.

Reply:
Ok I indeed was misunderstanding how it typehinted kwargs, thanks for that info, I have modified it.